vyper/venom/venom_to_assembly.py

```diff
@@ -308,7 +314,7 @@ def _generate_evm_for_basicblock_r(
 
         ref.extend(asm)
 
-        for bb in basicblock.reachable:
+        for bb in basicblock.cfg_out:
```

Comment:
Could be reachable removed from basic block if it is no longer used here?

Reply:
Yes, good point

Reply:
https://github.com/vyperlang/vyper/pull/4330/commits/1b117c8e9f51701501fe0ba277d063d4bf4a4a69